server/web_ui/shared/static/shared.js

```diff
@@ -1,6 +1,52 @@
 // This is easier to have in JS than in WASM
 export function modal_hide_by_id(m) {
-    var elem = document.getElementById(m);
-    var modal = bootstrap.Modal.getInstance(elem);
+    const elem = document.getElementById(m);
+    const modal = bootstrap.Modal.getInstance(elem);
     modal.hide();
-};
\ No newline at end of file
+}
+
+function changeTag(node, newTag) {
```

Comment:
Please use rust function names/variables since that's what we're doing elsewhere for now 😄  (`change_tag` vs `changeTag`, `new_tag` vs `newTag`)... and now I think of it, that function's _replacing_ the tag so it would be better calling it `replace_tag`.